packages/client/src/components/UploadForm.tsx

```diff
@@ -0,0 +1,77 @@
+import { authFetch } from '@/helpers';
+import { AuthState } from '@/types';
+import { useState } from 'react';
+import { SignedUrlResponseBody } from '@guardian/transcription-service-common';
+
+const uploadToS3 = async (url: string, blob: Blob) => {
+	try {
+		const response = await fetch(url, {
+			method: 'PUT',
+			body: blob,
+		});
+		const status = await response.status;
```

Comment:
I _think_ that response.status should just be an integer not a promise https://developer.mozilla.org/en-US/docs/Web/API/Response/status

Reply:
oops - good call